cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,40 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
-  before(() => {
-    
-  });
 
-  it('', () => {
-    
+  const randRadio = 'gender-radio-' + Math.ceil(Math.random() * 3);
```

Comment:
```suggestion
  const randomRadio = 'gender-radio-' + Math.ceil(Math.random() * 3);
```
do not shorten the names